### ISSUE TITLE
Question: Item layouts support for RadListView in angular

### ISSUE THREAD

**Issue by shripalsoni04** [telerik/nativescript-ui-samples-angular#9]
I am not able to find any example or any documentation of using [item-layouts](http://docs.telerik.com/devtools/nativescript-ui/Controls/NativeScript/ListView/item-layouts) using angular. I also checked angular code in `listview-directive.ts`, but there by default `ListViewLinearLayout` is assigned to `listViewLayout` property, and I am not able to find any code to set `listViewLayout` there.

So my question is, different layouts are supported in angular version of RadListView or it is still under development?

**Comment by VladimirAmiorkov** [comment]
Hi @shripalsoni04 ,

Thank you for your question. In the current version (1.2.0) of the angular directives for RadListView the `ListViewLinearLayout` is not yet available. 

As Angular 2 is still in development (currently at RC4) we are actively working on being up-to date with it and its code bade while also bringing full support for all of the components in the [nativescript-telerik-ui](https://www.npmjs.com/package/nativescript-telerik-ui)/[nativescript-telerik-ui-pro](https://www.npmjs.com/package/nativescript-telerik-ui-pro) plugins. The directives for the RadListView layouts are currently being worked on and will be released with the next official version.

**Comment by VladimirAmiorkov** [comment]
Hi @shripalsoni04, 

In the current 1.2.0 version you can customize the `listViewLayout` by creating an **ListViewLinearLayout** object in the code behind (Component) and pass it to the RadListView's `listViewLayout` via Angular 2 binding. Like so, note that this code snippet is for example purpose and will not work out of the box. You will need to setup the items of the RadListBox, itemTemplate etc.:

```
import { Component, OnInit } from "@angular/core";
import * as listViewModule from "nativescript-telerik-ui-pro";

@Component({
    selector: "my-app",
    template: `
         <RadListView [listViewLayout]="myLayout" >
         </RadListView>
    `
})
export class AppComponent implements OnInit {
    private _myLayout: listViewModule.ListViewLinearLayout;

    constructor() {
   
    }

    ngOnInit() {
        this._myLayout = new listViewModule.ListViewLinearLayout();
        this._myLayout.scrollDirection = "Horizontal";
    }
    get myLayout(): listViewModule.ListViewLinearLayout {
        return this._myLayout;
    }
}
```

**Comment by VladimirAmiorkov** [comment]
Implemented in version 1.3.1 of [nativescript-telerik-ui](https://www.npmjs.com/package/nativescript-telerik-ui)/[nativescript-telerik-ui-pro](https://www.npmjs.com/package/nativescript-telerik-ui-pro)
